.mergify.yml

```diff
@@ -1,26 +1,46 @@
+queue_rules:
+  - name: default
+    conditions:
```

Comment:
if they are identical to condition to be put in queue you don't need to repeat them fwiw.